frontend/sass/_data-driven-onboarding.scss

```diff
@@ -2,6 +2,17 @@
     margin-top: 2.25em;
 }
 
+.level {
```

Comment:
If this isn't a default Bulma class, can you prefix it with `jf` so it's called `jf-level`?  That way it will be easier for readers to know which classes are ours vs. which are Bulma's.

Reply:
so believe it or not `level` is indeed a default Bulma class: https://bulma.io/documentation/layout/level/. Described as a multipurpose horizontal container.

Reply:
Oh my bad!  That's cool!